spread.yaml

```diff
@@ -556,7 +556,7 @@ prepare: |
     fi
     if [[ "$SPREAD_SYSTEM" == opensuse-* ]]; then
         # refresh metadatadata
-        zypper ref
+        zypper --no-gpg-checks ref
```

Comment:
I assume this is the actual workaround for the issues? Can you add a comment explaining what's the issue we are seeing with gpg checks?

Reply:
sure, thanks for the review